docs/documentation/securing_apps/topics/oidc/oauth21-support.adoc

```diff
@@ -12,6 +12,6 @@ side may need to be still done manually or through some other third-party soluti
 ==== OAuth 2.1 client profiles
 
 To make sure that your clients are OAuth 2.1 compliant, you can configure Client Policies in your realm as described in the link:{adminguide_link}#_client_policies[{adminguide_name}]
-and link them to the global client profiles for OAuth 2.1 support, which are automatically available in each realm. You can use `oauth-2-1-for-confidential-client` profile for confidential clients.
+and link them to the global client profiles for OAuth 2.1 support, which are automatically available in each realm. You can use ether `oauth-2-1-for-confidential-client` profile for confidential clients or `oauth-2-1-for-public-client` profile for public clients.
```

Comment:
```suggestion
and link them to the global client profiles for OAuth 2.1 support, which are automatically available in each realm. You can use either `oauth-2-1-for-confidential-client` profile for confidential clients or `oauth-2-1-for-public-client` profile for public clients.
```

Small typo. Do you please have a chance to fix it?

Reply:
Thank you. I fixed it.